libs/contact/guides/drivers/hubspot-forums-driver.md

```diff
@@ -0,0 +1,21 @@
+# Hubspot Forums Driver
+
+The Hubspot Forums Driver allows your Daffodil Contact Forum feature to connect directly to your hubspot account to manage your contact forum submissions. 
+
+To set up, inside of your `app.module` first import `DaffContactHubSpotDriverModule` from `@daffodil/contact`. Next, in your imports section include `DaffContactHubSpotDriverModule.forRoot(config)`, where config is an object containing information needed to connect to your hubspot forum. This object should contain a `portalId`, `guid`, and can optionally contain a `version`. [Follow this link on where to find your forum's information](https://knowledge.hubspot.com/forms/find-your-form-guid).
```

Comment:
So it is typed as a `DaffHubspotConfig` but do you think it's better to just say the type to be brief or list out all the things explicitly needed.

Reply:
If the model changes or something, we'll need to come here and change the docs (or more likely, we'll forget and the docs will be wrong). Presumably, at some point we'll have api docs for all the models and such so this will be autogenerated and easily accessible.